src/session/sidebar/row.rs

```diff
@@ -489,6 +507,75 @@ fn stringify_message(message: Message) -> String {
                 }
             }
         }
+        MessageContent::MessageChatChangePhoto(_) => {
+            show_sender = false;
+            match message.chat().type_().clone() {
+                ChatType::Supergroup(data) if data.is_channel() => gettext("Channel photo changed"),
+                _ => {
+                    if message.is_outgoing() {
+                        gettext("You changed group photo")
+                    } else {
+                        gettext!(
+                            "{} changed group photo",
+                            sender_name(message.sender(), true)
+                        )
+                    }
+                }
+            }
+        }
+        MessageContent::MessagePinMessage(data) => {
+            show_sender = false;
+            let message_content = match message.chat().history().message_by_id(data.message_id) {
+                Some(data) => match data.content().0 {
+                    MessageContent::MessageText(data) => {
+                        let msg = data.text.text;
+                        if msg.chars().count() > 17 {
+                            gettext!(" «{}...»", msg.chars().take(16).collect::<String>())
```

Comment:
I would rather use https://www.compart.com/de/unicode/U+2026 for the 3 dots. @melix99 ?
```suggestion
                            gettext!(" «{}…»", msg.chars().take(16).collect::<String>())
```